src/common/fluentUIThemes/FluentUIDefaultTheme.ts

```diff
@@ -0,0 +1,15 @@
+import { createTheme, getTheme, ITheme } from "office-ui-fabric-react/lib/Styling";
+
+export const fluentUIDefaultTheme = (): ITheme => {
+    let currentTheme;
+    const themeColorsFromWindow: any = (window as any).__themeState__.theme;
+    if (themeColorsFromWindow) {
+        currentTheme = createTheme({
+            palette: themeColorsFromWindow
+        });
+    }
+    else
+        currentTheme = getTheme();
+
+    return currentTheme;
```

Comment:
This seems to be the same method currently present in `src/controls/adaptiveCardHost/fluentUI/FluentUI.tsx`. Any chance you could double-check and remove the duplication if appropriate?

Reply:
Yes, I have a plan to consolidate all this stuff into the common folder... I must do PR for the AdaptiveCardHost.
Basically, this is the first step to consolidate the same themes into the common folder, then I'll use the same code from common folder.